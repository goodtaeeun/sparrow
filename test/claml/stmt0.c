int main() {
  int x = 1, y = 0;
  while (x < 10) {
    y++;
  }
  return x + 1;
}
