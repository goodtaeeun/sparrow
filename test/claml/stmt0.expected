/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 1 "stmt0.c"
int main(void) {
  int x;
  int y;
  int __cil_tmp3;

  {
#line 2
    x = 1;
#line 2
    y = 0;
    {
#line 3
      while (1) {
      while_continue: /* CIL Label */;
#line 3
        if (!(x < 10)) {
#line 3
          goto while_break;
        }
#line 4
        y++;
      }
    while_break: /* CIL Label */;
    }
#line 6
    return (x + 1);
  }
}
